packages/evolution-common/src/utils/__tests__/ParamsValidatorUtils.test.ts

```diff
@@ -0,0 +1,606 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+import { ParamsValidatorUtils } from '../ParamsValidatorUtils';
+import { v4 as uuidV4 } from 'uuid';
+
+class TestClass { }
+
+describe('ParamsValidatorUtils', () => {
+
+    describe('isObject', () => {
+        test('should return no errors for a valid object', () => {
+            const errors = ParamsValidatorUtils.isObject('attr', {}, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
```

Comment:
maybe a test with `null` and real non-empty object?

Reply:
added